org.gnome.Cheese.appdata.xml.in.patch.1

```diff
@@ -0,0 +1,27 @@
+--- a/data/org.gnome.Cheese.appdata.xml.in
++++ b/data/org.gnome.Cheese.appdata.xml.in
+@@ -37,4 +37,24 @@
+   <update_contact>davidk@gnome.org</update_contact>
+   <project_group>GNOME</project_group>
+   <translation type="gettext">cheese</translation>
++  <content_rating type="oars-1.1"/>
++  <releases>
++    <release version="3.34.0" date="2019-09-10">
++      <description>
++        <p>
++          <ul>
++            <li>
++              build: Don't use 'lib' prefix in find_library
```

Comment:
This is still a truely awful changelog for users. If they have no release notes for this release maybe just leave it empty and add one for 3.32.x or something. Surely this application has had some update with real changes.

Reply:
Another attempt in 08b0dd3 let me know if this is more workable.

Reply:
Looks good